js/src/pages/price-benchmark/banner/index.scss

```diff
@@ -0,0 +1,66 @@
+.gla-price-benchmark-suggestions-banner__body {
+	align-items: start;
+	display: grid;
+	grid-template-columns: 164px 1fr;
+	gap: $grid-unit-20;
+	padding: $grid-unit-40 $grid-unit-30;
+
+	.gla-price-benchmark-suggestions-banner__image_container {
+		grid-row: span 3;
+		place-self: center center;
+	}
+
+	.gla-price-benchmark-suggestions-banner__badge {
+		background-color: #f6f7f7;
+		border: none;
+		border-radius: 2px;
+	}
+
+	.gla-price-benchmark-suggestions-banner__title {
+		margin: $grid-unit-15 0 0 0;
+	}
+
+	.gla-price-benchmark-suggestions-banner__description {
+		grid-column: 2 / -1;
+		margin: 0;
+	}
+
+	.gla-price-benchmark-suggestions-banner__dismiss-button {
+		grid-column: 2 / -1;
+		justify-self: start;
+	}
+
+	@media (min-width: $break-medium + 1px) {
+		.lower-description-text {
+			display: block;
+		}
+	}
+}
+
+@media (max-width: $break-medium) {
```

Comment:
@ankitrox For maintenance and ease to read through the code, media queries should not contain selectors. It should be the other way around, i.e selectors which contain media queries. It's easier to see in one place, i.e the selector, what's happening across different breakpoints.